server/src/test/java/org/apache/druid/server/coordinator/rules/PeriodLoadRuleTest.java

```diff
@@ -108,29 +111,61 @@ public void testAppliesToPartialOverlap()
   }
 
   @Test
-  public void testSerdeNullTieredReplicants() throws Exception
+  public void testIncludeFuture()
+  {
+    DateTime now = DateTimes.of("2012-12-31T01:00:00");
+    PeriodLoadRule includeFutureRule = new PeriodLoadRule(
+        new Period("P2D"),
+        true,
+        ImmutableMap.of("", 0)
+    );
+    PeriodLoadRule notIncludeFutureRule = new PeriodLoadRule(
+        new Period("P2D"),
+        false,
+        ImmutableMap.of("", 0)
+    );
+
+    Assert.assertTrue(
+        includeFutureRule.appliesTo(
+            builder.interval(new Interval(now.plusDays(1), now.plusDays(2))).build(),
+            now
+        )
+    );
+    Assert.assertFalse(
+        notIncludeFutureRule.appliesTo(
+            builder.interval(new Interval(now.plusDays(1), now.plusDays(2))).build(),
+            now
+        )
+    );
+  }
+
+  @Test
+  public void testSerdeNull() throws Exception
```

Comment:
Why did you change the test name?

Reply:
Because I have also test `Serde` null of `includeFuture` in this test case.

Reply:
Ok. Then, please add some comments about what null variables are being tested in this test. Same for the below test.